webview-ui/src/utils/context-mentions.ts

```diff
@@ -210,6 +211,42 @@ export function getContextMenuOptions(
 		}
 	}
 
+	if (dynamicSearchResults.length > 0) {
+		// Convert search results to queryItems format
+		const searchResultItems = dynamicSearchResults.map((result) => {
+			let formattedPath = ""
+			if (os === "windows") {
```

Comment:
Do we need to do this for the opened tabs too?

Reply:
@mrubens let me check it

Reply:
update pr for this :(